product-specifications/example-3.md

```diff
@@ -0,0 +1,47 @@
+# Project Specification: 'Maze Runner' CLI Game 🏃
+
+## Overview
+
+'Maze Runner' is a text-based, Command-Line Interface (CLI) game designed for users who appreciate the simplicity and challenge of classic text-based games. The main objective is to navigate a maze and find the exit as quickly as possible.
+
+## Goals
+
+1. To provide an engaging gaming experience to users who enjoy text-based games.
+2. To challenge players with randomly generated mazes of varying complexity.
+3. To include a scoring system based on maze complexity and completion time.
+
+## Scope
+
+The game will consist of the following components:
+
+1. Maze Generation: Rules for generating mazes of different sizes and complexity.
+2. User Interface: Text-based instructions and input prompts for player actions.
+3. Scoring System: Based on maze complexity and completion time.
+
```

Comment:
Is it possible to add a high score feature so players can go back to previous mazes and try to beat their times

Reply:
This would be fun, I think would make sense for the next version, but great idea!